ci/plugins/mzcompose/hooks/pre-exit

```diff
@@ -38,6 +60,18 @@ buildkite-agent artifact upload ps-aux.log
 docker ps -a --no-trunc > docker-ps-a.log
 buildkite-agent artifact upload docker-ps-a.log
 
+if grep "error: missing record 0: Record {" services.log; then
```

Comment:
This one would not be in services.log, but in the `run.log`. Therefore, it is important to fish for interesting lines in all logs.

An example where an error can switch from one log file to another would be your own https://github.com/MaterializeInc/materialize/issues/17616 -- it is now a panic, so is in `services.log`, but if Alexander converts it into a SQL error to be reported to the client, it will end up in the `run.log`